### PR TITLE
Adds Robert Summers

### LINKED ISSUE #793: Robert Summers
Add profile to website

### PR DESCRIPTION
**Pull request by nikeshashar**
Closes #793 

@alexandergimson - ready to queue up 
@anitacanita - 👀 

Merci

### CODE REVIEW

File: data/graduates.yml [hunk 1]
```diff
@@ -29,6 +29,36 @@
...
+  categories:
+    -remote
+    -acast
+    -career-changer
+  case_study:
+   - question: What were you doing before joining Makers Academy?
+     answer: I was working as an insurance technician.
+   - question: How did you come across Makers Academy and what made you decide to apply?
+     answer: I was researching how to make my career change to software development when Makers came up on a google search.  I decided on joining Makers after reading reviews from former students, knowing that Makers has a vested interested in my success through placement with hiring partners, and doing the Remote taster day.
+   - question: How did you find learning remotley?
```
Comment (anitacanita) [inline data/graduates.yml]: *remotely

File: data/graduates.yml [hunk 2]
```diff
@@ -29,6 +29,36 @@
...
+     answer: Learning remotely was great. I didn't loose any time to a 'commute' and the technology for remote pairing made it feel natural, like really working next to my pair programmer for the day.
+   - question: What was your end of cohort project?
+     answer: We made an android app in Java - a language we had to learn specifically for the project. The app makes running a game by giving the user 'coins' based on the distance run. It saves the high score and plays different music if a new high score is recorded.
+   - question: How has your life changed since learning to code?
+     answer: I find myself excited to get up and go to work in the morning, which never happened when I worked in insurance. I'm proud of what I do and love being a Makers alumni I get to tell people about my story and how I got to where am now.
```
Comment (anitacanita) [inline data/graduates.yml]: I feel there's a missing comma somewhere?

File: data/graduates.yml [hunk 3]
```diff
@@ -29,6 +29,36 @@
...
+   - question: What advice would you give to people thinking of changing career?
+     answer: Do as much research as you can and try to get into the 'community' of your new career (like going to meetups etc.)  Try to spend your spare time doing as much of the 'free' things like online courses, meetups, and hackathons as possible to get a feel for the new career. If it feels right, then go for it!
+   - question: What would you say to people who say £4000 is expensive for an online course?
+     answer: The price seems expensive, but you really get a lot out of the course.  The quality of the course is worth it in itself.  The career support Makers provide after graduation is extremely valuable.  The comradery and friendships you make with the students, alumni, and staff is priceless.
```
Comment (anitacanita) [inline data/graduates.yml]: Seems to have extra spaces after every full stop.
Should you leave the last comma in "The comradery and friendships you make with the students, alumni, and staff is priceless."?

File: data/graduates.yml [hunk 4]
```diff
@@ -29,6 +29,36 @@
...
+   - question: What was your biggest fear before joining MA and how do you feel about it now?
+     answer: My biggest fear was that employers wouldn't think a 'bootcamp' grad would be good enough to hire. I now know this is not true. When you meet real developers, you realize that the course has given you the ability to 'talk the talk' with them.  You soon come to understand that many developers appreciate the practical experience you gain at Makers over the theoretical knowledge you would have gained from a CS degree.
```
Comment (anitacanita) [inline data/graduates.yml]: Extra space after the last full stop.

**Comment by nikeshashar** [comment]
@anitacanita made some changes. Can I haz :shipit:

**Pull request merged**